src/core/Track.cpp

```diff
@@ -1707,29 +1707,6 @@ void TrackOperationsWidget::clearTrack()
 
 
 
-/*! \brief Create and assign a new FX Channel for this track */
-void TrackOperationsWidget::createFxLine()
-{
-	int channelIndex = gui->fxMixerView()->addNewChannel();
-
-	Engine::fxMixer()->effectChannel( channelIndex )->m_name = m_trackView->getTrack()->name();
-
-	assignFxLine(channelIndex);
-}
-
-
-
-/*! \brief Assign a specific FX Channel for this track */
-void TrackOperationsWidget::assignFxLine(int channelIndex)
-{
-	Track * track = m_trackView->getTrack();
-	dynamic_cast<InstrumentTrack *>( track )->effectChannelModel()->setValue( channelIndex );
-
-	gui->fxMixerView()->setCurrentFxLine( channelIndex );
-}
-
-
-
```

Comment:
Are you removing the logic from the track here in favor of the instrument?  (if so, why not both?)